docs/release_notes/0.0.27.md

```diff
@@ -11,7 +11,9 @@ By using the following commands, one can now create a cluster using a declaratio
 `okctl apply cluster` applies a cluster declaration that defines what parts of the cluster should exist
```

Comment:
Unrelated to this code line:

> This code breaks client / api pattern. I need to go back and rewrite
some things later. Delete cluster needs to get done so, so it takes
priority over correctness

As long as we agree that we actually will follow the client/api pattern later, and that later doesn't mean too long down the road (not sure which "deadline" which is good), I agree with the get-it-out-now decision.